dts/bindings/serial/atmel,sam0-uart.yaml

```diff
@@ -27,12 +27,19 @@ properties:
       required: true
       description: Transmit Data Pinout
 
-    rxdma:
-      type: int
-      required: false
-      description: Receive DMA channel
-
-    txdma:
-      type: int
-      required: false
-      description: Transmit DMA channel
+    dmas:
+      description: |
+        Optional TX & RX dma specifiers.  Each specifier will have a phandle
+        reference to the dmac controller, the channel number, and peripheral
+        trigger source.
+
+        For example dmas for TX, RX on SERCOM3
+           dmas = <&dmac 0 0xb>, <&dmac 0 0xa>;
```

Comment:
I think important add MCU model when given example. For SAMR21 SERCOM3 triggers are 0x8 and 0x7.
Not sure about dma channels, but I believe should be 1 and 0, for instance.

For example the MCU xxxx dmas for TX, RX on SERCOM3
dmas = <&dmac 1 0xb>, <&dmac 0 0xa>;